hubspot.com.hubspot-template-3.json

```diff
@@ -4,7 +4,6 @@
     "serviceId": "hubspot-template-3",
     "serviceName": "HubSpot CMS Site",
     "version": 1,
-    "syncBlock": true,
```

Comment:
The template in this form would not be secure if `syncBlock` is removed.
If you intend to use signing, please add  `syncPubKeyDomain` in the same PR.
You may also want to add `syncRedirectDomain` for the sync flow, if you plan to use redirects.

Reply:
Thanks Pawel! I just updated it so this should be good now. Let me know if you have any more concerns!